integrations/quickbooks/syncs/general-ledger.ts

```diff
@@ -0,0 +1,24 @@
+import type { QuickBooksJournalEntry } from '../types';
+import type { NangoSync } from '../../models';
+import { paginate } from '../helpers/paginate.js';
+import type { PaginationParams } from '../helpers/paginate';
+import { mapQuickBooksToUnified } from '../mappers/toLedger.js';
```

Comment:
```suggestion
import { mapQuickBooksToUnified } from '../mappers/to-ledger.js';
```
and change the file name

Reply:
updated this and and others in the quickbooks directory to kebab-case ✔